### PR TITLE
fixed the issue where confirming a reviewed sighting wasn't redirecti…

### PR DESCRIPTION
**Pull request by emilypatsko**
…ng (plus some minor styling fixes)

### CODE REVIEW

File: Controllers/AdminController.cs [hunk 1]
```diff
@@ -33,16 +33,10 @@ public Sighting ConfirmSighting([FromRoute] int id)
...
-            {
-                return BadRequest(ModelState);
-            }
-
-            _sightings.UpdateAndConfirmSighting(SightingToUpdate);
-
-            return StatusCode(200);
+            var sightingUpdated =_sightings.UpdateAndConfirmSighting(SightingToUpdate);
+            return sightingUpdated;
         }
```
Comment (chloejonessoftwire) [inline Controllers/AdminController.cs:49]: looks good to me!

File: ClientApp/src/NavBar/NavBar.scss [hunk 2]
```diff
@@ -80,6 +80,7 @@ li a:hover {
   right: 0;
   color: $light-text;
   z-index: 1001;
+  background-color: transparent !important;
```
Comment (ChristianMcc) [inline ClientApp/src/NavBar/NavBar.scss]: can we find a way to do this without using !important keyword? take a look at https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity on how to do this and maybe a warning on not using important :)

File: ClientApp/src/Pages/ConfirmSighting/ConfirmSighting.tsx [hunk 3]
```diff
@@ -41,8 +41,10 @@ export function ConfirmSightingForm(): JSX.Element {
 
   function confirmOrDeleteSighting(event: FormEvent) {
     event.preventDefault();
-    if (button == "confirm") {
+    if (button == "confirm") 
+    {
       setStatus("SUBMITTING");
+      console.log(status);
```
Comment (ChristianMcc) [inline ClientApp/src/Pages/ConfirmSighting/ConfirmSighting.tsx]: can delete this :)

File: ClientApp/src/Pages/ConfirmSighting/ConfirmSighting.tsx [hunk 4]
```diff
@@ -58,7 +60,10 @@ export function ConfirmSightingForm(): JSX.Element {
       })
         .then(() => setStatus("FINISHED"))
         .catch(() => setStatus("ERROR"));
-    } else if (button == "delete") {
+      console.log(status);
```
Comment (ChristianMcc) [inline ClientApp/src/Pages/ConfirmSighting/ConfirmSighting.tsx]: and here

**Review by ChristianMcc — COMMENTED** [review]
Looking good! Couple of small points on css and console.logs :)

**Review by littlemauli — COMMENTED** [review]
looks good to me

**Review by ChristianMcc — APPROVED** [review]

File: Controllers/AdminController.cs [hunk 5]
```diff
@@ -33,16 +33,10 @@ public Sighting ConfirmSighting([FromRoute] int id)
...
 
         [HttpPost("updateAndConfirmSighting/{id}")]
-        public IActionResult updateAndConfirmSighting([FromBody] Sighting SightingToUpdate)
+        public Sighting updateAndConfirmSighting([FromBody] Sighting SightingToUpdate)
         {
-            if (!ModelState.IsValid)
```
Comment (ChristianMcc) [inline Controllers/AdminController.cs:38]: how come we removed this? Is there not some validation on sighting that might be useful?

**Review by ChristianMcc — CHANGES_REQUESTED** [review]
Just a thought about modelstate being removed before we merge actually!

File: Controllers/AdminController.cs (hunk 5 above)
Comment (emilypatsko) [reply Controllers/AdminController.cs:38]: fixed it!

**Review by ChristianMcc — APPROVED** [review]

**Pull request merged**